package.json

```diff
@@ -28,6 +28,7 @@
     "url": "https://github.com/agenda/agenda/issues"
   },
   "dependencies": {
+    "debug": "^2.6.8",
```

Comment:
Actually since we require it in the file it'll need to be in deps otherwise we'll get issues it being missing. Ignore that.